bin/www

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
```

Comment:
Express generator doing this :p . I think renaming by adding the .js extension is a good idea, will do that.